src/content/upgrades/merge/eth-issuance/index.md

```diff
@@ -0,0 +1,100 @@
+
+Issuance of new ETH on this chain is rewarded to validators for properly attesting to the state of the chain, and is also rewarded when a validator proposes a block. Rewards (or penalties) are calculated and distributed every epoch (6.4 minutes) based on validator performance for that time period. These rewards are _significantly_ lower than the 2 ETH being issued under proof-of-work today every ~13.5 seconds, as operating a validating node is not an economically intense activity, thus does not require or warrant as high a reward.
+
+### Pre-merge issuance breakdown {#pre-merge-issuance-breakdown}
+
+Total ETH supply: **~119,300,000 ETH** (as of Q2 2022)
+
+**Execution layer issuance:** <br/>
+
+- Estimating at 2.08 ETH per 13.3 seconds\*: **~4,930,000** ETH issued in a year
+- Currently inflating at **~4.13%** (4.93M per year / 119.3M total)
+- \*This includes the 2 ETH per canonical block, plus an average of 0.08 ETH over time from ommer blocks. Also uses 13.3 second, the baseline block time target without any influence from a [difficulty bomb](/glossary/#difficulty-bomb). ([See source](https://bitinfocharts.com/ethereum/))
+
+**Consensus layer issuance:**
+
+- Using 13,000,000 total ETH staked, the rate of ETH issuance is ~1600 ETH/day ([See source](https://ultrasound.money/))
+- Results in **~584,000** ETH issued in a year
+- Currently inflating at **~0.49%** (584K per year / 119.3M total)
+
+<InfoBanner>
+<strong>Total annual issuance rate: ~4.62%</strong> (4.13% + 0.49%)
+
+**~89.4%** of the issuance is going to miners on the execution layer (4.13 / 4.62 \* 100)
+
+**~10.6%** is being issued to stakers on the consensus layer (0.49 / 4.62 \* 100)
+</InfoBanner>
+
+## Post-merge {#post-merge}
+
+### Execution layer issuance {#el-issuance-post-merge}
+
+Will be zero. Proof-of-work will no longer be valid under the rules of consensus, and all execution layer activity will be included in "beacon blocks" which are published and attested to by proof-of-stake validators.
+
+### Consensus layer issuance {#cl-issuance-post-merge}
+
+Will continue as it was, with small rewards for attesting-to and proposing blocks. These rewards will continue to accrue to _validator balances_ that are managed within the consensus layer. These are separate Ethereum accounts to the accounts we're used to on Mainnet, and until the Shanghai upgrade funds from validator accounts will not be withdrawable/transferrable. This means that although new ETH is still being issued, 100% of it will be locked from the market until this upgrade occurs. When the Shanghai upgrade is rolled out, this ETH will become available.
```

Comment:
```suggestion
Consensus layer issuance will continue as before The Merge, with small rewards for validators who attest to and propose blocks. Validator rewards will continue to accrue to _validator balances_ that get managed within the consensus layer. These are separate Ethereum accounts to the accounts we're used to on Mainnet, and until the Shanghai upgrade funds from validator accounts will not be withdrawable/transferrable. This means that although new ETH is still being issued, 100% of it will be locked from the market until this upgrade occurs. When the Shanghai upgrade is rolled out, this ETH will become available.
```